draft-ietf-quic-transport.md

```diff
@@ -1438,6 +1438,51 @@ actual exponent is one-less than the explicit exponent, and the value represents
 to 0xFFFF.
 
 
+### ACK Frames and Packet Protection
+
+ACK frames that acknowledge protected packets MUST be carried in a packet that
+has an equivalent or greater level of packet protection.
+
+Packets that are protected with 1-RTT keys MUST be acknowledged in packets that
+are also protected with 1-RTT keys.
+
+A packet that is not protected and claims to acknowledge a packet number that
+was sent with packet protection is not valid.  An unprotected packet that
+carries acknowledgments for protected packets MUST be discarded in its entirety.
```

Comment:
In practice, discarding previously processed frames in a packet once an invalid ack is discovered is somewhat difficult.  I think it's the right suggestion, but I worry it won't be implemented correctly.